src/languages/css.js

```diff
@@ -25,6 +25,10 @@ export default function(hljs) {
     hljs.APOS_STRING_MODE,
     hljs.QUOTE_STRING_MODE
   ];
+  const CSS_VARIABLE = {
+    className: "variable",
+    begin: /--[A-Za-z_-](?:(?!\s)[A-Za-z0-9_-])*/,
```

Comment:
This looks a bit overly complex a regex... what are we trying to accomplish with the `(?!\s)`? Must variables be two letters at least?

Reply:
> Must variables be two letters at least?

No, there is no such constraint.

I think you are right about the complexity of the regex. Maybe we could start with a simpler regex for now. 

What do you think of this?
```suggestion
    begin: /--[A-Za-z][A-Za-z0-9_-]*/,
```

Reply:
That looks much more like what I would have expected to see.